internal/characters/tartaglia/burst.go

```diff
@@ -19,19 +19,18 @@ const (
 
 func init() {
 	// burst (melee) -> x
-	burstMeleeFrames = frames.InitAbilSlice(102)
-	burstMeleeFrames[action.ActionAttack] = 101
-	burstMeleeFrames[action.ActionSkill] = 102
-	burstMeleeFrames[action.ActionDash] = 102
-	burstMeleeFrames[action.ActionJump] = 102
-	burstMeleeFrames[action.ActionSwap] = 100
+	burstMeleeFrames = frames.InitAbilSlice(103) // Q -> D
+	burstMeleeFrames[action.ActionAttack] = 102  // Q -> N1
+	burstMeleeFrames[action.ActionSkill] = 102   // Q -> E
+	burstMeleeFrames[action.ActionDash] = 102    // Q -> D
```

Comment:
I believe this should be the jump action.
```suggestion
	burstMeleeFrames[action.ActionJump] = 102    // Q -> J
```